mmhuman3d/core/visualization/renderer/torch3d_renderer/lights.py

```diff
@@ -0,0 +1,99 @@
+from typing import Union
+
+import torch
+from pytorch3d.renderer.lighting import AmbientLights as _AmbientLights
+from pytorch3d.renderer.lighting import DirectionalLights as _DirectionalLights
+from pytorch3d.renderer.lighting import PointLights as _PointLights
+from pytorch3d.renderer.utils import TensorProperties
+
+
+class MMLights(TensorProperties):
+
+    def __init__(self, **kwargs) -> None:
+        super().__init__(**kwargs)
+        _N = 1
+        for k in [
+                'ambient_color', 'diffuse_color', 'specular_color', 'location',
+                'direction'
+        ]:
+            if hasattr(self, k):
+
+                v = getattr(self, k)
+                if not isinstance(v, torch.Tensor):
+                    v = torch.Tensor(v)
+                v = v.view(-1, 3)
+                setattr(self, k, v)
+
+                if getattr(self, k).shape[0] > _N:
+                    _N = getattr(self, k).shape[0]
+        for k in [
```

Comment:
The list is used more than once. Assign a costant value like `MMLIGHT_ATTR` looks better.

Reply:
ok